starter-code/src/app/my-home/my-home.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
```

Comment:
una vez arreglada la parte del import e inyectando el servicio podías hacer algo así: 
`<ul *ngFor="let movie of movieArray"><p>{{movie.title}}</p></ul>`